TASVideos/Pages/Shared/_UserFileInfo.cshtml

```diff
@@ -75,14 +75,16 @@
 		</div>
 	</cardbody>
 </div>
+
 <div condition="!string.IsNullOrWhiteSpace(Model.Annotations)" class="mb-2">
-	<card>
+	<div class="card border-primary-subtle border mb-2">
 		<cardheader><h4>Annotations</h4></cardheader>
 		<cardbody>
-			@Model.Annotations
+			<pre style="white-space: pre-wrap">@Model.Annotations</pre>
```

Comment:
please do not use inline styles.  If bootstrap doens't have a style for this, add a class and put the style in a style tag

Reply:
> add a class and put the style in a style tag

How?